Makefile

```diff
@@ -217,7 +217,7 @@ tools: # Install dependencies and tools required to build
 
 .PHONY: test
 test: # Run tests
-	go test ./...
+	go test ./... -race
```

Comment:
We could also through `$(TESTARGS)` in here while we're at it, so we can easily run individual/filtered tests like the make step below